src/hooks.ts

```diff
@@ -31,13 +39,43 @@ const usePlaybackStateWithoutInitialValue = () => {
     };
   }, []);
 
-  return state;
+  return playWhenReady;
 };
 
-/** Get current playback state and subsequent updates  */
-export const usePlaybackState = () => {
-  const state = usePlaybackStateWithoutInitialValue();
-  return state ?? State.None;
+/**
+ * Get current playback state and subsequent updates.
+ *
+ * Note: While it is fetching the initial state from the native module, the
+ * returned state property will be `undefined`.
+ * */
+export const usePlaybackState = (): PlaybackState | { state: undefined } => {
```

Comment:
Why the merge? Having the hooks in seperate files does feel tidier. And cleaner once we start adding unit tests. I'm all for smaller & simpler files.